app/views/reviews/edit.html.erb

```diff
@@ -0,0 +1,13 @@
+<h3> Edit Review</h3>
+
+<%= form_for :review, url: donut_review_path(@donut, @review), method: "patch" do |f| %>
+  <%= f.label :rating %>
+  <%= f.collection_radio_buttons :rating, [[1, "1"], [2, "2"], [3,"3"], [4,"4"],[5,"5"]], :first, :last %>
```

Comment:
In the future I would make sure to define these options in a method on the review model:`[[1, "1"], [2, "2"], [3,"3"], [4,"4"],[5,"5"]]`, so you are not duplicating this in multiple views. 😃  something like `def review_rating_options`